### ISSUE TITLE
MA0051 counts empty lines

### ISSUE THREAD

**Issue by hankovich** [meziantou/Meziantou.Analyzer#576]
### Version of the Meziantou.Analyzer NuGet package

2.0.71

### Rule Identifier

MA0051

### Target Framework

net8.0

### C# Language version

C# 12

### Description

MA0051 counts empty lines, so it's reported for short methods with a lot amount of short local functions

### Reproduction Steps

Minimal code:

```c#
public static class MyClass
{
    public static void Foo() // MA0051: Method is too long (102 lines; maximum allowed: 60)
    {
        Local1();

        static void Local1()
        {
            Local2();
        }

        static void Local2()
        {
            Local3();
        }

        static void Local3()
        {
            Local4();
        }

        static void Local4()
        {
            Local5();
        }

        static void Local5()
        {
            Local6();
        }

        static void Local6()
        {
            Local7();
        }

        static void Local7()
        {
            Local8();
        }

        static void Local8()
        {
            Local9();
        }

        static void Local9()
        {
            Local10();
        }

        static void Local10()
        {
            Local11();
        }

        static void Local11()
        {
            Local12();
        }

        static void Local12()
        {
            Local13();
        }

        static void Local13()
        {
            Local14();
        }

        static void Local14()
        {
            Local15();
        }

        static void Local15()
        {
            Local16();
        }

        static void Local16()
        {
            Local17();
        }

        static void Local17()
        {
            Local18();
        }

        static void Local18()
        {
            Local19();
        }

        static void Local19()
        {
            Local20();
        }

        static void Local20()
        {
            Local21();
        }

        static void Local21()
        {
            Local22();
        }

        static void Local22()
        {
            Local23();
        }

        static void Local23()
        {
            Local24();
        }

        static void Local24()
        {
            Local25();
        }

        static void Local25()
        {
            Local26();
        }

        static void Local26()
        {
            Local27();
        }

        static void Local27()
        {
            Local28();
        }

        static void Local28()
        {
            Local29();
        }

        static void Local29()
        {
            Local30();
        }

        static void Local30()
        {
            Local31();
        }

        static void Local31()
        {
            Local32();
        }

        static void Local32()
        {
            Local33();
        }

        static void Local33()
        {
            Local34();
        }

        static void Local34()
        {
            Local35();
        }

        static void Local35()
        {
            Local36();
        }

        static void Local36()
        {
            Local37();
        }

        static void Local37()
        {
            Local38();
        }

        static void Local38()
        {
            Local39();
        }

        static void Local39()
        {
            Local40();
        }

        static void Local40()
        {
            Local41();
        }

        static void Local41()
        {
            Local42();
        }

        static void Local42()
        {
            Local43();
        }

        static void Local43()
        {
            Local44();
        }

        static void Local44()
        {
            Local45();
        }

        static void Local45()
        {
            Local46();
        }

        static void Local46()
        {
            Local47();
        }

        static void Local47()
        {
            Local48();
        }

        static void Local48()
        {
            Local49();
        }

        static void Local49()
        {
            Local50();
        }

        static void Local50()
        {
            // nothing
        }
    }
}
```

### Other information

As you see, it says method has 102 lines, so it looks like some additional lines are counted. As I think `Foo` has only 1 line, not 102.

**Comment by hankovich** [comment]
I have `MA0051.skip_local_functions = true` in my `.editorconfig`

**Comment by meziantou** [comment]
Leading and trailing trivia (comments / spaces) are not counted yet. #581 will also exclude trivia.
